Routing/Loader/DirectoryRouteLoader.php

```diff
@@ -34,6 +37,10 @@ public function __construct(RestRouteProcessor $processor)
      */
     public function load($resource, $type = null)
     {
+        if (isset($resource[0]) && '@' === $resource[0]) {
+            $resource = $this->fileLocator->locate($resource);
```

Comment:
Not sure if that makes sense. I mean what advantage do we have by passing everything to the file locator?